stub-idp/src/main/java/uk/gov/ida/stub/idp/domain/factories/StubCoreTransformersFactory.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.ida.stub.idp.domain.factories;
+
+import org.opensaml.xmlsec.algorithm.DigestAlgorithm;
+import org.opensaml.xmlsec.algorithm.SignatureAlgorithm;
+import uk.gov.ida.saml.core.transformers.outbound.decorators.ResponseAssertionSigner;
+import uk.gov.ida.saml.core.transformers.outbound.decorators.ResponseSignatureCreator;
+import uk.gov.ida.saml.core.transformers.outbound.decorators.SamlResponseAssertionEncrypter;
+import uk.gov.ida.saml.core.transformers.outbound.decorators.SamlSignatureSigner;
+import uk.gov.ida.saml.security.EncrypterFactory;
+import uk.gov.ida.saml.security.EncryptionKeyStore;
+import uk.gov.ida.saml.security.EntityToEncryptForLocator;
+import uk.gov.ida.saml.security.IdaKeyStore;
+import uk.gov.ida.saml.security.IdaKeyStoreCredentialRetriever;
+import uk.gov.ida.saml.security.KeyStoreBackedEncryptionCredentialResolver;
+import uk.gov.ida.saml.security.SignatureFactory;
+import uk.gov.ida.saml.security.SignatureWithKeyInfoFactory;
+import uk.gov.ida.saml.serializers.XmlObjectToBase64EncodedStringTransformer;
+import uk.gov.ida.stub.idp.transformers.UnsignedAssertionCapableResponseToSignedStringTransformer;
+
+/**
+ * Replaces <i>some</i> calls that would have been made to CoreTransformersFactory in saml-libs - in particular,
+ * provides a ResponseStringTransformer that can optionally skip signing of assertions: essential for stub countries
+ * (but not much else).
+ */
+public class StubCoreTransformersFactory {
+
+    static UnsignedAssertionCapableResponseToSignedStringTransformer getResponseStringTransformer(
+        final EncryptionKeyStore publicKeyStore,
+        final IdaKeyStore keyStore,
+        final EntityToEncryptForLocator entityToEncryptForLocator,
+        final SignatureAlgorithm signatureAlgorithm,
+        final DigestAlgorithm digestAlgorithm,
+        final boolean signAssertions) {
+        return getResponseStringTransformer(publicKeyStore, keyStore, entityToEncryptForLocator, signatureAlgorithm,
+            digestAlgorithm, new EncrypterFactory(), signAssertions);
+    }
+
+    public static UnsignedAssertionCapableResponseToSignedStringTransformer getResponseStringTransformer(
```

Comment:
This seems to be the only public method signature that's actually being used, are the three useful?